_notes.md

```diff
@@ -1,12 +1,17 @@
-
-- [X] Use Celery to run requested workflows
-- [ ] Download requested files from the Internet using `requests.get()`
-- [ ] Run CWL conformance tests via WES
-  - [ ] Make a CLI entry to submit workflows for `cwltest`?
-- [ ] Add option to auto start server to `toil launch-cluster`
-- [ ] Should also automate creation of security group rule if this option is set
-- [ ] Add some custom Toil endpoints to fetch files via HTTP
-  - [ ] The stdout/stderr in the log are supposed to be URLs pointing to the real file, not the infinitely long contents
-- [ ] Update docs
+- [X] Use Celery to run workflows
+- [X] Support workflow files from the Internet (using `requests.get()`)
+- [ ] Add custom Toil endpoints to serve static log/output files?
+  - [ ] The stdout/stderr files are supposed to be URLs pointing to the real file, not the infinitely long contents
+- Usability
+  - [ ] Add option to auto start server to `toil launch-cluster`
+  - [ ] Script to prepare celery (start rabbitmq, celery workers, etc.)
+- Improve documentation
   - [ ] Add details on how to pass toil specific parameters to `POST /runs`
+  - [ ] Only recommend users to open the TCP port to the public (until we have SSL/TLS support?)
+- Testing
+  - [ ] Run CWL conformance tests via WES
+    - [ ] Make a CLI entry to submit workflows for `cwltest`?
```

Comment:
There's a `cwl-runner`[https://w3id.org/cwl/v1.2/cwl-runner.cwl] compatible `wes-client` in https://github.com/common-workflow-language/workflow-service

Reply:
Oh cool! I can use that instead.

Although, I was having some issues with including files defined in the CWL file for the tests. For example, for [this workflow](https://github.com/common-workflow-language/common-workflow-language/blob/main/v1.0/v1.0/template-tool.cwl#L7), I couldn't find a programatic way to attach `underscore.js` to the WES run request..